### DISCUSSION TITLE
请求co-maintainer权限

### DISCUSSION THREAD

**Discussion by glyh** [Yuukiy/JavSP#287] [General]
这个项目虽然很实用但是可以提升的空间太多了，近期有空可能会发很多PR，我感觉如果有comaintainer权限可能会方便一些。

**Comment by glyh** [comment]
@Yuukiy 请问意向如何？

**Reply by Yuukiy** [reply]
@glyh 已邀请。请保持master分支的提交简洁，尽可能一个commit一件事，并且具有有意义的commit message

**Reply by glyh** [reply]
感谢

**Reply by glyh** [reply]
请问可以说明下新的改动应该放到哪个branch里以及release流程之类的吗

**Reply by Yuukiy** [reply]
> 请问可以说明下新的改动应该放到哪个branch里以及release流程之类的吗

就前面说的，保持master提交简洁就好，对开发分支没有什么要求。release的话我来吧，主要是尽可能确保配置文件的稳定，减小版本间的不兼容问题。

**Reply by Yuukiy** [reply]
新的提交看起来变化很大，我把之前的版本按1.8发布了，这也是我又上线时的原版计划版本。
新的预计按2.0来发布。

**Reply by glyh** [reply]
好的，已经更新[Changelog](https://github.com/Yuukiy/JavSP/blob/master/CHANGELOG.md)
